pkg/kubelet/apis/config/scheme/testdata/KubeletConfiguration/after/v1beta1.yaml

```diff
@@ -53,7 +53,7 @@ kind: KubeletConfiguration
 kubeAPIBurst: 10
 kubeAPIQPS: 5
 logging:
-  flushFrequency: 5000000000
+  flushFrequency: 5s
```

Comment:
Because the field is used in a beta API, there are beta stability expectations for this field.